v2/tools/asoctl/pkg/crd/crd.go

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package crd
+
+import "github.com/spf13/cobra"
+
+// NewCommand creates a new cobra to clean deprecated crd versions
+func NewCommand() (*cobra.Command, error) {
+
```

Comment:
minor: You have an extra newline here but not for some other functions. I don't mind the newline at the start but I think we should be consistent. Most of your functions don't have a newline here so maybe remove it?